### PR TITLE
Fixes #52 - Requirements for the testsuite

### LINKED ISSUE #52: Define TS prerequisites
Define TS prerequisites and document them in README

### PR DESCRIPTION
**Pull request by rsvoboda**
Fixes #52 - Requirements for the testsuite

Please make sure your PR meets the following requirements:
- [x] Pull Request contains description for the changes
- [x] Pull Request does not include fixes for multiple issues / topics

### CODE REVIEW

**Comment by rsvoboda** [comment]
Travis issue

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (rename-unpacked-jboss-as) on project microprofile-test-suite: Execution rename-unpacked-jboss-as of goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run failed: Plugin org.apache.maven.plugins:maven-antrun-plugin:1.8 or one of its dependencies could not be resolved: Could not transfer artifact org.apache.ant:ant:jar:1.9.4 from/to google-maven-central (https://maven-central.storage-download.googleapis.com/maven2/): GET request of: org/apache/ant/ant/1.9.4/ant-1.9.4.jar from google-maven-central failed: Connection reset -> [Help 1]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # eap-microprofile-test-suite
  a small standalone test suite for MicroProfile on WF/EAP
 
+## Requirements for the testsuite
+ - JDK 8+
+ - Docker
+ 
+ Maven is not a hard requirement as `mvnw` is delivered with this repository.
```
Comment (mnovak1) [inline README.md]: `Maven is not a hard requirement as ...` however maven is not mentioned among above requirements at all. It should be listed there as well.

Comment (rsvoboda) [reply README.md]: mvnw is in the repo, Maven is not really required

Comment (rsvoboda) [reply README.md]: Standalone Maven on the system ...

Comment (rsvoboda) [reply README.md]: If the main concern is that README shows mvn commands and not ./mvnw commands please create new PR / issue to fix this concern

Comment (mnovak1) [reply README.md]: Yes, it's about recommended way how the test suite should be run. If using `mvn ...` command then Maven should be added as requirement.  If using `mvnw ...` command then Maven can be omitted.

Comment (rsvoboda) [reply README.md]: ok, here we go https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues/55
Task for tomorrow :)

**Pull request merged**